DiamondLightSource/mx-bluesky
# Merge deployment scripts into one

One script to deploy them all. Closes #532 


To test:
- Run test deployment and check that it complete without errors.
- Check docs are make sense.
